src/main/java/com/faforever/moderatorclient/api/domain/VotingService.java

```diff
@@ -117,7 +120,7 @@ public VotingQuestion create(VotingQuestion votingQuestion) {
     //region choices
     public List<VotingChoice> getAllChoicesFromApi() {
         log.debug("Retrieving all choices");
-        List<VotingChoice> result = fafApi.getAll(ElideNavigator.of(VotingChoice.class).collection());
+        List<VotingChoice> result = fafApi.getAll(ElideNavigator.of(VotingChoice.class).collection().addIncludeOnCollection("votingQuestion"));
```

Comment:
Please follow the style as in line 79: if there are more operations put them in new lines.